client/main.lua

```diff
@@ -129,23 +129,27 @@ local function openCanteen()
 	exports.ox_inventory:openInventory('shop', { type = 'Canteen', id = 1})
 end
 
-local function spawnNPCsIfNotExisting()
-	if DoesEntityExist(canteenPed) or DoesEntityExist(freedomPed) then return end
+local function pedCreate(pedModel, x, y, z, heading, scenario)
```

Comment:
parameter should take in a vector4 instead of x, y, z, heading, and then expand it into the CreatePed native